lib/install/validate/validator.go

```diff
@@ -361,6 +351,63 @@ func (v *Validator) managedbyVC(ctx context.Context) {
 	return
 }
 
+func (v *Validator) credentials(ctx context.Context, input *data.Data, conf *config.VirtualContainerHostConfigSpec) {
+	if input.OpsPassword == nil {
```

Comment:
Could you add a comment for that please? just to avoid a confusion for a person who modify that code next time.